src/modules/roc_netio/target_uv/roc_netio/udp_receiver_port.cpp

```diff
@@ -263,5 +275,39 @@ void UDPReceiverPort::recv_cb_(uv_udp_t* handle,
     self.writer_.write(pp);
 }
 
+bool UDPReceiverPort::join_multicast_group_() {
+    if (!address_.miface()) {
+        return true;
+    }
+
+    if (int err = uv_udp_set_membership(&handle_, packet::ip_to_str(address_).c_str(),
+                                        address_.miface(), UV_JOIN_GROUP)) {
+        roc_log(LogError, "udp receiver: uv_udp_set_membership(): [%s] %s",
+                uv_err_name(err), uv_strerror(err));
+        return false;
+    }
+
+    roc_log(LogDebug, "udp receiver: join multicast group: address=%s interface=%s",
+            packet::address_to_str(address_).c_str(), address_.miface());
```

Comment:
Does address_to_str() also formats the `@...` part? If it does we don't need to explicitly print interface here. If it doesn't, I guess we should fix it?

Reply:
> Does address_to_str() also formats the @... part?

It does not. I fixed it. I think we should use `ip@iface` instead of `ip@iface:port` here.

Reply:
Agree.

Reply:
But I'd expand the log message to make things explicit, something like:

```
udp receiver: joined multicast group %s at interface %s
```